tests/internal/test_database_monitoring.py

```diff
@@ -152,12 +152,46 @@ def test_dbm_dddbs_peer_service_enabled():
     sqlcomment = dbm_popagator._get_dbm_comment(dbspan_with_peer_service)
     # assert tags sqlcomment contains the correct value
     assert (
-        sqlcomment
-        == "/*dddbs='db-name-test',dde='staging',ddps='orders-app',ddpv='v7343437-d7ac743',traceparent='%s'*/ "
-        % (dbspan_with_peer_service.context._traceparent,)
+        sqlcomment == "/*dddb='db-name-test',dddbs='db-name-test',dde='staging',ddps='orders-app',"
+        "ddpv='v7343437-d7ac743',traceparent='%s'*/ " % (dbspan_with_peer_service.context._traceparent,)
     ), sqlcomment
 
 
+@pytest.mark.subprocess(
+    env=dict(
+        DD_DBM_PROPAGATION_MODE="full",
+        DD_SERVICE="orders-app",
+        DD_ENV="staging",
+        DD_VERSION="v7343437-d7ac743",
+    )
+)
+def test_dbm_peer_entity_tags():
+    from ddtrace import tracer
+    from ddtrace.propagation import _database_monitoring
+
+    dbspan = tracer.trace("dbname")
+    dbspan.set_tag("peer.hostname", "some-hostname")
+    dbspan.set_tag("db.instance", "some-db-instance")
+
+    # when dbm propagation mode is full sql comments should be generated with dbm tags and traceparent keys
+    dbm_propagator = _database_monitoring._DBM_Propagator(0, "procedure")
+    sqlcomment = dbm_propagator._get_dbm_comment(dbspan)
+    # assert tags sqlcomment contains the correct value
+    assert (
```

Comment:
nit: I'd test using `ddh='some-hostname' in sqlcomment` (etc) just to avoid future refactorings if more/less tags are added

Reply:
thanks for the suggestion, makes sense but I decided to keep it this way to stay consistent with the existing tests